modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectParser.java

```diff
@@ -1039,24 +1054,25 @@ private Expression string(String context, Object obj, String fallback) {
     public Expression string(JSONArray json, int index) {
```

Comment:
Thanks for the changes here, I was planning on changing to literals from string values when renaming. Looks good.